test/run_test.py

```diff
@@ -15,7 +17,14 @@
 from torch.utils import cpp_extension
 from torch.testing._internal.common_utils import TEST_WITH_ROCM, shell, set_cwd, FILE_SCHEMA
 import torch.distributed as dist
-from typing import Dict, Optional
+from typing import Dict, Optional, Tuple, List
+
+try:
+    import boto3  # type: ignore[import]
+    import botocore  # type: ignore[import]
+    HAVE_BOTO3 = True
```

Comment:
i dont think this is used.

Reply:
Oh hm @samestep was there a reason for using HAVE_BOTO3 in the other scripts?/Is there something I should be doing with it here as well?

Reply:
I copied the convention from `HAVE_MYPY` (from `test/test_type_hints.py`)

Reply:
if you don't need the variable then you can just `pass` in the `except` branch (like I do in `torch/testing/_internal/mypy_wrapper.py`)

Reply:
I think this is a good (and mypy friendly convention)